### PR TITLE
Additional requirements for support for up to 4 BrickPi3 

### PR DESCRIPTION
**Pull request by omartin2010**
Additional requirements for support for up to 4 BrickPi3

### CODE REVIEW

File: ev3dev2/motor.py [hunk 1]
```diff
@@ -63,7 +63,10 @@
     from ev3dev2._platform.brickpi import OUTPUT_A, OUTPUT_B, OUTPUT_C, OUTPUT_D
 
 elif platform == 'brickpi3':
-    from ev3dev2._platform.brickpi3 import OUTPUT_A, OUTPUT_B, OUTPUT_C, OUTPUT_D
+    from ev3dev2._platform.brickpi3 import OUTPUT_A, OUTPUT_B, OUTPUT_C, OUTPUT_D, 
+                                           OUTPUT_E, OUTPUT_F, OUTPUT_G, OUTPUT_H,
```
Comment (WasabiFan) [inline ev3dev2/motor.py]: I think you need to escape the newline

Comment (omartin2010) [reply ev3dev2/motor.py]: I don't in my python code I'll do it here.

**Review by omartin2010 — COMMENTED** [review]
Modified the suggested patch.

**Review by WasabiFan — APPROVED** [review]
Imports work on my machine. I don't even have one BrickPi to actually test the ports, but as long as I can import it I'm happy.

**Comment by WasabiFan** [comment]
Thanks!

**Pull request merged**